### PR TITLE
Inject bintray.com/mozilla-appservices as the top repository

### PR DESCRIPTION
**Pull request by grigoryk**
Two repositories are now injected.
Plugin version is bumped to 0.3

### CODE REVIEW

**Comment by mcomella** [comment]
I'll get to this tomorrow.

**Comment by ncalexan** [comment]
I've been thinking about this, and I think it's better to have `maven.mozilla.org` and then `bintray.com/mozilla-appservices`.  That will let us push selected revisions to `m.m.o` to eat traffic if we need to, and it's more future-proof: we want `m.m.o` to serve everything _eventually_.

@grigoryk, am I missing some interaction here?

File: gradle-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Changed
 
+## [0.3] - 2019-03-22
```
Comment (mcomella) [inline gradle-plugin/CHANGELOG.md:10]: Note to self: we'll need to update this.

**Review by mcomella — APPROVED** [review]
lgtm.

@grigoryk Should I land this or should I postpone on Nick's comment? I'm missing context on whether or not something needs to change.

**Comment by ncalexan** [comment]
> @grigoryk Should I land this or should I postpone on Nick's comment? I'm missing context on whether or not something needs to change.

@grigoryk can you wire in `maven.mozilla.org` first, ASAP?  Our first bintray bill was much larger than anticipated so we need to divert traffic very quickly.

**Comment by ncalexan** [comment]
After this lands, we need to ensure that the following repos are pulling from `maven.mozilla.org` before `bintray.com/mozilla-appservices`:

- [ ] https://github.com/mozilla-mobile/android-components
- [ ] https://github.com/mozilla-mobile/fenix
- [ ] https://github.com/mozilla-mobile/reference-browser
- [ ] https://github.com/mozilla-lockbox/lockbox-android

Anything else?

**Review by mcomella — APPROVED** [review]
Updated version lgtm.

**Pull request closed without merging**